doc/nrf/config_and_build/bootloaders/bootloader_external_flash.rst

```diff
@@ -13,14 +13,19 @@ When using MCUboot, you can store the storage partition for the secondary slot i
 * Single-byte read and write.
```

Comment:
This is likely not the PR for it.
But should not the page on external flash with MCUboot mention CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY?

Reply:
That is a good point, I will see if I can address it here.

Reply:
With the newest commit, we no longer mention CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY at all.
Sorry for my somewhat circular argumentation, but while CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY is set by default if the DTS thing is on, it is still relevant to mention it, so users can disable it.
For example, I have seen users enabling external flash for something else, but  wanting MCUboot in internal flash.

Reply:
Apologies for the oversight, will address.